src/content/quickstart/openshift/ready_clusters.md

```diff
@@ -0,0 +1,27 @@
+### Make your clusters ready for submariner
+
+Submariner gateway nodes need to be able to accept traffic over ports 4500/UDP and 500/UDP
+when using IPSEC. In addition we use port 4800/UDP to encapsulate traffic from the worker nodes
+to the gateway nodes and ensuring that Pod IP addresses are preserved.
```

Comment:
same here ensuring->ensure

Use the bits from here: src/content/quickstart/openshiftgn/_index.md